packages/network/samba/config/smb.conf

```diff
@@ -49,6 +44,10 @@
   passdb backend = smbpasswd
   smb encrypt = disabled
   use sendfile = yes
+  # local master = yes
+  # preferred master = auto
+  # domain master = auto
+  # os level = 20
```

Comment:
If we go with the commented solution I suppose an extra comment "# The following are the default values, but included for informational purposes" (or words to that effect) might be useful so that someone in 6 months time doesn't submit a PR removing them because they're the default values and therefore unnecessary etc. etc.